src/android/app/src/main/java/io/github/lime3ds/android/display/PortraitScreenLayout.kt

```diff
@@ -0,0 +1,17 @@
+// Copyright 2023 Citra Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+package io.github.lime3ds.android.display
+
+enum class PortraitScreenLayout(val int: Int) {
+    // These must match what is defined in src/common/settings.h
+    TOP_FULL_WIDTH(0),
+    CUSTOM_PORTRAIT_LAYOUT(1);
+
+    companion object {
+        fun from(int: Int): PortraitScreenLayout {
+            return entries.firstOrNull { it.int == int } ?: TOP_FULL_WIDTH;
+        }
+    }
+}
```

Comment:
Why is this its own file? The code here is seemingly only used by `fragments/EmulationFragment.kt`.

Reply:
I made this its own file to match with the structure of ScreenLayout.kt in the same folder, which I believe is also only used by EmulationFragment. I honestly am not sure what motivated that structure originally, but it seemed prudent to keep the same structure that already existed.

Reply:
I went ahead and moved this enum into the already existing ScreenLayout.kt file and deleted this file. Didn't seem to hurt anything, and it still makes sense to me.